test/plugin/scenarios/webflux-scenario/config/expectedData.yaml

```diff
@@ -14,30 +14,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 registryItems:
   applications:
-    - {webflux-projectB-scenario: nq 0}
-    - {webflux-projectA-scenario: nq 0}
+    - {webflux-scenario: nq 0}
   instances:
-    - {webflux-projectB-scenario: 1}
-    - {webflux-projectA-scenario: 1}
+    - {webflux-scenario: nq 0}
   operationNames:
-    - webflux-projectB-scenario: [org.apache.skywalking.apm.testcase.sc.webflux.projectB.controller.TestAnnotationController.success,
-                                  org.apache.skywalking.apm.testcase.sc.webflux.projectB.controller.TestAnnotationController.hello,
-                                  WEBFLUX.handle, org.apache.skywalking.apm.testcase.sc.webflux.projectB.controller.TestAnnotationController.error,
-                                  org.apache.skywalking.apm.testcase.sc.webflux.projectB.controller.TestAnnotationController.healthCheck,
-                                  RoutingConfiguration.org.apache.skywalking.apm.testcase.sc.webflux.projectB.route.TestHandler]
-    - webflux-projectA-scenario: [/testcase/annotation/mono/hello, /projectA/testcase,
-                                  /testcase/annotation/success, /testcase/annotation/healthCheck, /notFound, /projectA/healthCheck,
-                                  /testcase/route/error, /testcase/route/success, /testcase/annotation/error]
+    - webflux-scenario: ['WEBFLUX.handle']
   heartbeat: []
 segmentItems:
-  - applicationCode: webflux-projectB-scenario
+  - applicationCode: webflux-scenario
     segmentSize: nq 0
     segments:
       - segmentId: not null
         spans:
-          - operationName: org.apache.skywalking.apm.testcase.sc.webflux.projectB.controller.TestAnnotationController.success
+          - operationName: WEBFLUX.handle
```

Comment:
Why removed the class name as operation name? Could you explain?

Reply:
By using the class name(old one), it is not as clear as URL, but at least, it could separate different logic. Today, you posted here are all the same operation names. This plugin is becoming useless.